irods/test/connection_test.py

```diff
@@ -159,8 +160,18 @@ def test_assigning_session_connection_timeout_to_invalid_values__issue_569(self)
             with self.assertRaises(ValueError):
                 sess.connection_timeout = value
 
-        # Test previously set value is unaffected
-        self.assertEqual(sess.connection_timeout, DESIRED_TIMEOUT)
+    def test_assigning_session_connection_timeout_to_large_values__issue_623(self):
+        # Test use of a too-large timeout in iRODSSession constructor as well as on assignment to the
+        # iRODSSession property 'connection_timeout'.  In both cases, error checking and hard-limiting
+        # should be immediate.
+        sess = helpers.make_session(connection_timeout = MAXIMUM_CONNECTION_TIMEOUT + 1)
+        self.assertEqual(sess.connection_timeout, MAXIMUM_CONNECTION_TIMEOUT)
+
+        sess.connection_timeout = MAXIMUM_CONNECTION_TIMEOUT + 1
+        self.assertEqual(sess.connection_timeout, MAXIMUM_CONNECTION_TIMEOUT)
+
+        coll = helpers.home_collection(sess)
+        sess.collections.get(coll)
```

Comment:
what do these do?

Reply:
This is just there to use connection and prove it's still  viable.  It is probably irrelevant now that the timeout takes immediate effect in the value of the connection timeout attribute, rather than showing up lazily in the act of using the connection.  Will delete....